Packages/com.unity.inputsystem/InputSystem/Editor/InputParameterEditor.cs

```diff
@@ -182,6 +182,17 @@ internal override void SetTarget(object target)
             OnEnable();
         }
 
+#if UNITY_INPUT_SYSTEM_UI_TK_ASSET_EDITOR
+        /// <summary>
+        /// Default stub implementation of <see cref="InputParameterEditor.OnDrawVisualElements"/>.
+        /// Should be overridden to create the desired UI.
+        /// </summary>
+        public override void OnDrawVisualElements(VisualElement root, Action onChangedCallback)
```

Comment:
Great that you figured out a solution to the XR toolkit issue we had with the v1.6.0 release